src/Views/batchprocessor.html

```diff
@@ -7,7 +37,7 @@
     <h4 class="modal-title">StreamStats Batch Processor</h4>
 </div>
 <div class="modal-body">
-    <ul id="aboutTabs" class="nav nav-tabs">
+    <ul id="bpTabs" class="nav nav-tabs">
         <li ng-class="{'active':vm.selectedBatchProcessorTabName == 'submitBatch'}"><a
                 ng-click="vm.selectBatchProcessorTab('submitBatch')"><i class="fa fa-question-circle"></i>&nbsp;Submit

```

Comment:
I know this wasn't part of this issue, but can we change the icon from `fa-question-circle` to something more relevant? Or feel free to remove both icons for the tabs if you think that works better.

Reply:
Great catch! I'm gonna try and find some new ones, see if they are better liked.